csharp/extractor/Semmle.Extraction.CIL/Entities/NoMetadataHandleType.cs

```diff
@@ -100,11 +107,7 @@ public override IEnumerable<IExtractionProduct> Contents
 
         public override string Name => GenericsHelper.GetNonGenericName(name);
 
-        public override Namespace? ContainingNamespace => isContainerNamespace
-            ? containerName == Cx.GlobalNamespace.Name
-                ? Cx.GlobalNamespace
-                : new Namespace(Cx, containerName)
-            : null;
```

Comment:
This property was always generating a new `Namespace`. Also, `SystemNamespace` could also be reused.